cves/CVE-2017-11719.yml

```
@@ -357,11 +405,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 125
 CWE_note:
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it. If the
   report mentions a nickname, e.g. "Heartbleed", use that. Or come up with one!
 
   Must be under 30 characters. Optional. Be appropriate.
-nickname:
+nickname: "Foul Ball Scan"
```

```suggestion
nickname: Foul Ball Scan
```
Same thing here & for the other single quoted lines.